snuba/cli/devserver.py

```diff
@@ -152,6 +152,17 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                     "--consumer-group=metrics_group",
                 ],
             ),
+            (
+                "metrics-direct-write-consumer",
+                [
+                    "snuba",
+                    "multistorage-consumer",
```

Comment:
You don't need multistorage-consumer if you are writing to a singe storage. The regular consumer should work.

Reply:
Sure, though eventually this will write to 3 tables.